akka-actor/src/main/resources/reference.conf

```diff
@@ -918,9 +931,14 @@ akka {
         # Must implement akka.io.DnsProvider
         provider-object = "akka.io.InetAddressDnsProvider"
 
-        # These TTLs are set to default java 6 values
-        positive-ttl = 30s
-        negative-ttl = 10s
+        # To set the time to cache name resolutions
+        # Possible values:
+        # default: sun.net.InetAddressCachePolicy.get() and getNegative()
+        # forever: cache forever
+        # never: no caching
+        # n: positive timeout
```

Comment:
It's not only `n` but `n [time unit]`, so maybe say "positive timeout with unit, for example "30 s".